Resolve the compilation error of libmpeg2 (embedded in hypseus) when cross-compiling

Signed-off-by: Laurent Merckx <laurent-merckx@skynet.be>

--- a/src/cmake/modules/BuildLibMPEG2.cmake	2018-04-24 17:55:39.575154215 +0200
+++ b/src/cmake/modules/BuildLibMPEG2.cmake	2018-04-24 18:02:08.074948502 +0200
@@ -1,6 +1,6 @@
 macro( build_libmpeg2 )
 
-if( CMAKE_CROSS_COMPILING )
+if( CMAKE_CROSSCOMPILING )
     string( REGEX MATCH "([-A-Za-z0-9\\._]+)-(gcc|cc)$" RESULT ${CMAKE_C_COMPILER} )
     string( REGEX REPLACE "-(gcc|cc)$" "" RESULT ${RESULT} )
     set( CONFIGURE_ARGS "--host=${RESULT}" )
@@ -8,7 +8,8 @@ endif()
 
 externalproject_add( libmpeg2
 	PREFIX ${CMAKE_CURRENT_BINARY_DIR}/3rdparty
-	URL https://github.com/btolab/libmpeg2/archive/master.zip
+	GIT_REPOSITORY https://github.com/btolab/libmpeg2.git
+	GIT_TAG master
 	CONFIGURE_COMMAND autoreconf -f -i && <SOURCE_DIR>/configure ${CONFIGURE_ARGS} --prefix=${CMAKE_CURRENT_BINARY_DIR}/3rdparty --disable-shared --enable-static --disable-sdl
 	BUILD_IN_SOURCE 1
 	BUILD_COMMAND make V=0
